x-pack/test/apm_api_integration/basic/tests/metrics_charts/metrics_charts.ts

```diff
@@ -0,0 +1,455 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from '@kbn/expect';
+import { first } from 'lodash';
+import { FtrProviderContext } from '../../../common/ftr_provider_context';
+import { expectSnapshot } from '../../../common/match_snapshot';
+
+interface Chart {
+  title: string;
+  key: string;
+  yUnit: string;
+  noHits: boolean;
+  series: Array<{
+    title: string;
+    key: string;
+    type: string;
+    color: string;
+    overallValue: number;
+    data: Array<{ x: number; y: number | null }>;
+  }>;
+}
```

Comment:
nit: Don't we have this type in APM? Would be great if we can avoid duplicating it.

Reply:
https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/server/lib/metrics/transform_metrics_chart.ts#L12-L14

Reply:
https://github.com/elastic/kibana/blob/5be36b799d000af609934bd5ebc8fcbd24c8c96d/x-pack/plugins/apm/server/lib/metrics/transform_metrics_chart.ts#L12-L14